openlibrary/templates/type/edition/view.html

```diff
@@ -493,7 +493,9 @@ <h4>$_("Excerpts")</h4>
                       $else:
                         $_("added anonymously.")
                       $if excerpt.comment:
-                        "<em>$excerpt.comment</em>"
+                        <div class="excerpt-comment">
+                          $:format(excerpt.comment)
+                        </div>
```

Comment:
```suggestion
                        $:format(excerpt.comment)
```
This div doesn't seem to serve a purpose.